string ProgramLocation
string ProgramName
string Comment
bool Upload
string IpAdress
geometry_msgs/Pose[] RobotPoses
bool[] PointColorViz
int32[] IndexVector
---
bool ReturnStatus
string ReturnMessage
